### DISCUSSION TITLE
LiDAR data points

### DISCUSSION THREAD

**Discussion by Robatic2021** [FieldRobotEvent/Virtual_Field_Robot_Event#10] [General]
Dear all,

We, the Robatic team 2021, have observed that, in comparison with earlier years, the maize plants are located on a larger distance from eachother (50 centimetres instead of 13 centimetres). To us, this seems a bit unrealistic, as maize plants on "real" fields are planted on a distance of 13 centimetres. 

On the robot, this results in the fact that the LiDAR will recieve less data points than in the previous years. The result is a harder, and perhaps less accurate, navigation through the field. We thus would like to know wether the intra-row distance of the simulated maize field will be decreased to a more realistic value, providing more data points.

Thanks in advance,
The Robatic team 2021

**Comment by Darkproduct** [comment]
Hi, we (Kamaro) are currently working on version 2 of the virtual maize field generator and this and some other issues will be addressed there. 

The new default distance between maize plants will be a bounded Gaussian distribution with μ = 16 cm and σ = 3 cm bounded between 13 and 19 cm.

**Reply by Robatic2021** [reply]
Hello, thanks for the reply! We are looking forward to the new generator. Will the world generator include stones and tasks 3/4, so generate for example beer cans and weeds? And when can the world generator be expected?

**Reply by Darkproduct** [reply]
Hi, yes I'm working on a version where you can generate worlds for task 1-4. I'm not sure if there'll be one big release (merge) or some smaller ones and I can't make any promises about a release date.

You can checkout our progress [here](https://github.com/Kamaro-Engineering/Virtual_Field_Robot_Event/tree/new-field-generator). Currently, the code looks bad and half of it doesn't work, but you can see the direction.
